README.md

```diff
@@ -21,27 +21,27 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+This is a migration file which will migrate new table data specified in the file to the database. The number is a timestamp which will migrate sequentially based on that timestamp.
 ```
 
 ### Question 3
 
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+The controllers actions are delegated by the router.
 ```
```

Comment:
:+1: although to be explicit, the router matches an incoming request with a controller action.